lib/dal/src/workspace_snapshot/graph/v4/tests.rs

```diff
@@ -3,7 +3,7 @@ use std::collections::HashMap;
 use si_events::{ulid::Ulid, ContentHash};
 
 use crate::{
-    workspace_snapshot::{graph::WorkspaceSnapshotGraphV3, node_weight::NodeWeight},
+    workspace_snapshot::{graph::WorkspaceSnapshotGraphV4, node_weight::NodeWeight},
```

Comment:
Do you think it would make sense to move these tests to a `graph/current/...`, and update them to use `WorkspaceSnapshotGraphVCurrent`?

There will always be things that are specific to whatever changes that were made for that graph version (which should probably live in `graph/v4/`), but a lot (if not all) of the tests in this area feel more suited for "graphs (in general) should behave like ...".